packages/tests/src/middleware/chaining.ts

```diff
@@ -0,0 +1,249 @@
+// import {
```

Comment:
Should we commit this yet?

Reply:
fixed in the next pr - apologies I didn't realise I had it here.